less/index.less

```diff
@@ -69,67 +70,84 @@ h1, h2, h3, h4, h5 {
     margin: auto;
 }
 
+
+/* Variables */
+
+@primary-body-size: 1.6rem;
+@primary-sub-heading-size: 1.8rem;
+
+
+/* Mixins */
+
+.body-borders(@color1, @color2) {
+    border-top: solid 1px black;
+    border-left: solid 1px @color1;
+    border-right: solid 1px @color2;
+}
+
+
 /* Header Styles */
 
 header {
     width: 100%;
     display: flex;
-}
 
-header nav {
-    margin-top: 40px; 
-    width: 610px;
-    justify-content: space-between;
-    display: flex;
-}
+    nav {
+        margin-top: 40px; 
+        width: 610px;
+        justify-content: space-between;
+        display: flex;
 
-header nav a {
-    text-decoration: none;
-    color: #b0b0b0;
-}
+        a {
+            text-decoration: none;
+            color: #b0b0b0;
+            font-size: @primary-body-size;
 
-header nav a:hover {
-    text-decoration: underline;
-    color: black;
-}
+            &:hover {
+                text-decoration: underline;
+                color: black;
+            }
+        }
+    }
 
-header .logo {
-    align-self: flex-end;
-    margin: 28px 0 0 91px;
+    .logo {
+        align-self: flex-end;
+        margin: 28px 0 0 91px;
+    }
 }
 
 /* Call To Action (CTA) Styles */
 
 .cta {
     display: flex;
-    margin-top: 72px;
+    margin-top: 7.2rem;
     justify-content: flex-end;
-}
 
-.cta .cta-text {
-    font-size: 72px;
-    text-align: center;
-    letter-spacing: 5px;
-    display: flex;
-    justify-content: center;
-    flex-flow: column;
-    align-items: center;
-    margin: 0 82px 0 0;
-}
-
-.cta .cta-text button {
-    border: 1px double black;
-    font-size: 16px;
-    background: white;
-    width: 170px;
-    height: 32px;
-    margin-top: 10px;
-    cursor: pointer;
-}
-
-.cta .cta-text button:hover {
-    background: black;
-    color: white;
+    .cta-text {
+        font-size: 72px;
+        text-align: center;
+        letter-spacing: 5px;
+        display: flex;
+        justify-content: center;
+        flex-flow: column;
+        align-items: center;
+        margin: 0 82px 0 0;
+
+        button {
```

Comment:
I think parametric mixins are awesome, and button creators are a great opportunity to use them so that some things stay the same (such as maybe the shape and size) while other things change (such as maybe the background color). Try making a button creator here.